src/components/hero/hero.scss

```diff
@@ -0,0 +1,22 @@
+.hero-body {
+  font-family: Titillium Web, sans-serif;
```

Comment:
If you do a rebase from the upstream repo, this PR https://github.com/code4romania/taskforce-fe-components/pull/21 adds these by default. We can also remove it from L17.
```suggestion
```